osquery/tables/yara/yara_utils.cpp

```diff
@@ -260,6 +260,9 @@ Status YARAConfigParserPlugin::setUp() {
 Status YARAConfigParserPlugin::update(const std::string& source,
                                       const ParserConfig& config) {
   // The YARA config parser requested the "yara" top-level key in the config.
+  if (config.count("yara") == 0) {
+    return Status(1, "No Yara Configuration");
```

Comment:
Should this be a `Status(1...`? I'm under the impression it's not an _error_ if the yara config doesn't load, but I'm also writing similar logic for this and want to be consistent.

Reply:
This is a side effect of the the JSON object removal above. I would agree though this is not an error.